### ISSUE TITLE
Not works on react-native [has solution]

### ISSUE THREAD

**Issue by ColCh** [kadirahq/storybook-addon-knobs#41]
Error text:
------------------
```
ExceptionsManager.js:61 Expected a component class, got [object Object].
```
![image](https://cloud.githubusercontent.com/assets/599439/18815273/c5472502-8333-11e6-80d9-e71d09af5e55.png)

WHY
-------------
Because of this wrapper:
https://github.com/kadirahq/storybook-addon-knobs/blob/7757f1ef03880f3682cc836f5e88f52078f4be51/src/components/WrapStory.js#L54
__WrapStory.js__:
```js
  render() {
    const { storyFn, context } = this.props;
    return <div>{storyFn(context)}</div>;
  }
```
`react-native` have no `<div>` component. It's only defined in WEB environment.

SOLUTION
----------------
remove wrapper and return storyFn result:
```diff
  render() {
    const { storyFn, context } = this.props;
-    return <div>{storyFn(context)}</div>;
+    return storyFn(context);
  }
```

**Comment by ColCh** [comment]
With fix applied works like a charm: https://streamable.com/esn8

**Comment by arunoda** [comment]
Okay. Looks prefect. Let's ship this.

**Comment by mnmtanish** [comment]
Released `v1.2.2`
